calculate_radius_max_wind

```diff
@@ -0,0 +1,60 @@
+def calculate_radius_max_wind(rho_air, latitude, max_wind_speed, central_pressure,
+                              ambient_pressure, radius_30kts, radius_50kts):
+    """
```

Comment:
You should still provide a brief description of what this is doing and perhaps some references to papers it is based on.  Also some of what the input is may be in a storm object which would shorten the number of input arguments.

Reply:
I seem to get what your mean.

Reply:
Ok, I got it!